websites/G/Genshin Impact Map/presence.ts

```diff
@@ -0,0 +1,171 @@
+const presence = new Presence({
+		clientId: "973729201104511007"
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000),
+	maps = [
```

Comment:
this array should be typed

Reply:
maps top browsingTimestamp ?

Reply:
what

Reply:
where array typed ?

Reply:
I’m saying the array should be typed with an interface along with the variables below